### ISSUE TITLE
Recaptcha is not diaplaying

### ISSUE THREAD

**Issue by karthigesh** [Cnordbo/RECaptcha-for-Codeigniter#16]
In controller i have used this:
$this->load->library('recaptcha');
$data['recaptcha_html'] = $this->recaptcha->recaptcha_get_html();
$this->load->view($data);

In view file i have used this:
<a href="<?php echo $this->recaptcha->recaptcha_get_signup_url(); ?>" >Get your API Code HERE</a></p>
<?php echo $recaptcha_html; ?>

![norecapcha](https://user-images.githubusercontent.com/22536708/27959726-34d40f46-6346-11e7-8e9d-e9e7fa5f24b8.png)

**Comment by ivan-yosifov** [comment]
Yeah, we have the same problem. Could it be because Google no longer supports the old reCAPTCHA v1?

**Comment by Cnordbo** [comment]
Sorry for the delayed response! This is an integration against V1 - I guess V2 could be easily adapted, altough this needs further investigation, which im unable to do atm.
